align_anything/evaluation/benchmarks/AGIEval/vllm_eval.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 PKU-Alignment Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import argparse
+import json
+from align_anything.evaluation.inference.base_inference import BaseInferencer_vllm
```

Comment:
Our base_inference file has been decoupled, try importing from vllm_inference?

Reply:
OK. I fix this problem in my code. But some of the old files in your repo remains this problem.
For example in _ARC/ds_infer.py_:
`from align_anything.evaluation.inference.ds_inference import BaseInferencer_deepspeed`

Reply:
You can test these new Benchmarks by reusing evaluate.sh script.

Reply:
> You can test these new Benchmarks by reusing evaluate.sh script.

I haven't set up the environment for alignment yet, so I suggest you test the new benchmark yourself, which will save time. And if you encounter any issues, I can make modifications based on the error type or you could directly modify it based on your need. 

However, if it's inconvenient for you, I can set up the environment tonight and test the code.

Reply:
It's convenient to set up the environment and conduct tests in our alignment environment, and you can also directly make changes to the code and commit them.

Reply:
OK. I'll try that.

Reply:
@ChangranXU hi, have you ever conducted tests in our alignment environment? I notice you modify agieval

Reply:
Yes, I set up the env and simply run the code. Initially it outputs with several errors and after modification, no error output. BUT, still the program seems only output the info, and can't run successfully locally, so i just stopped to try further. 😔

By the way, all vllm related files are provided by the Professor HAN, and according to the vllm codes, i created corresponding deepspeed files.

Reply:
Could you please let us know if you have resolved the bug you mentioned? If you need any assistance from our side, we are here to help.